js/views/modals/About/Donations.js

```diff
@@ -16,34 +18,27 @@ export default class extends baseVw {
 
   events() {
     return {
-      'click #copyAddress': 'copyDonationAddress',
-      'click #donationAddress': 'copyDonationAddress',
+      'click .js-copyAddress': 'copyDonationAddress',
     };
   }
 
   copyDonationAddress() {
-    const address = document.querySelector('#donationAddress');
-    const range = document.createRange();
-    const selection = window.getSelection();
-    range.selectNodeContents(address);
-    selection.removeAllRanges();
-    selection.addRange(range);
-
     clipboard.writeText(obDonationAddress);
 
-    this.$('#copyNotification').addClass('active');
+    this.$('.copyNotification').addClass('active');
```

Comment:
There's a certain convention we follow when targeting elements via a class from JS: we preface those classes with `js-`. The reasoning is that it's clear from looking at the markup that a particular element is being managed via JS. If we just used a non 'js' prefaced class, then if some css changed where that class wasn't needed, removing the class would break the JS link.

Similarly, if we're using a class for styling purposes, we don't preface it with 'js'.

So, if an element has both CSS and JS applied to it, it would have both a js and non-js class (e.g. `js-copyNotification` & `copyNotification`).

Sorry for the confusion on this.